src/lando/main/auth.py

```diff
@@ -73,6 +77,30 @@ def wrapper(request, *args, **kwargs):
     return wrapper
 
 
+def force_auth_refresh(f: Callable) -> Callable:
+    """
+    Decorator which forces authenticated session to be refreshed.
+    """
+
+    def wrapper(*args, **kwargs):
+        """Set oidc_id_token_expiration to 0, forcing session refresh."""
+        # First check that SessionRefresh is indeed enabled.
+        if "mozilla_django_oidc.middleware.SessionRefresh" not in settings.MIDDLEWARE:
+            raise RuntimeError("SessionRefresh middleware required but is not enabled.")
```

Comment:
Question: could we check this outside the wrapper, either in app setup or outside of `wrapper` at the start of `force_auth_refresh`? Seems this check will run on every call, when it's really a detail that we could assert once and assume elsewhere in the app.

Reply:
It might make sense to add a separate (app) check, to ensure that SessionRefresh middleware is enabled. However this is meant here as an integrated fail-safe, since there is a security component to this decorator and functionally it would result in insecure behaviour if it is used without SessionRefresh middleware being enabled.

I think it is cleaner to keep all functionality in the wrapper, which is pretty conventional for wrappers like this.